runner/src/main.rs

```diff
@@ -967,6 +968,24 @@ fn run_cargo_udeps() -> Step {
     }
 }
 
+fn run_cargo_clean() -> Step {
+    Step::Multiple {
+        name: "cargo clean".to_string(),
+        steps: crate_manifest_files()
+            .map(|manifest_path| {
+                let manifest_path = manifest_path.to_str().unwrap();
```

Comment:
You could first map `manifest_path` to string by calling (`to_string`). It is more idiomatic to do this using another call to map (`map(to_string)`). See https://github.com/project-oak/oak/blob/b599b42578d9af0f51a33dc4eb8f1a7f3c54c184/runner/src/main.rs#L906-L907

Reply:
done!